apina-core/src/main/kotlin/fi/evident/apina/model/type/ApiType.kt

```diff
@@ -1,21 +1,23 @@
 package fi.evident.apina.model.type
 
+import fi.evident.apina.model.settings.OptionalTypeMode
+
 sealed class ApiType {
 
-    abstract fun toTypeScript(): String
+    abstract fun toTypeScript(optionalTypeMode: OptionalTypeMode = OptionalTypeMode.NULL): String
```

Comment:
Perhaps we could leave the default value out? I guess we always want to provide a value to ensure that all code is translated consistently using same mode and a default value would just leave a possibility for bugs.

I'm half tempted to add a more general settings-object as a parameter, but since it would only contain this one flag, this is probably the right thing to do now. It's always possible to generalize this later, if necessary.

Reply:
Sounds reasonable, I'm gonna remove this default.